src/cursor/find_cursor.ts

```diff
@@ -155,6 +155,7 @@ export class FindCursor<TSchema = any> extends AbstractCursor<TSchema> {
 
   /** Execute the explain for the cursor */
   explain(): Promise<Document>;
+  /** @deprecated Callbacks are deprecated and will be removed in the next major version */
   explain(callback: Callback): void;
   explain(verbosity?: ExplainVerbosityLike): Promise<Document>;
   explain(
```

Comment:
We're missing a deprecation on the overload where explain has verbosity and a callback

Reply:
This is the function's defintion, there is no overload that accepts `[verbosity, callback]`

Reply:
see above